.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea/*
 *.iml
-target/*
+target/*
```

Comment:
please also add logs/ here so that logs aren't pushed to github and added to git history